src/pages/index.tsx

```diff
@@ -6,6 +6,7 @@ import useFetch from '@/hooks/useFetch';
 import classNames from '@/styles/tasks.module.scss';
 import task from '@/interfaces/task.type';
 import Accordion from '@/components/Accordion';
+// import { COMPLETED } from '@/components/constants/task-status'; unused variable.
```

Comment:
we can remove this comment if we are not using it

Reply:
For sure yes. I kept that there in case we needed it in the future 😅

Reply:
@Devashish0210 , you can resolve conversation if you have solved this